_validate_init.py

```diff
@@ -9,32 +9,32 @@
 import importlib
 
 
-components_package = 'dash_cytoscape'
+components_package = "dash_cytoscape"
```

Comment:
FWIW in other packages we've mostly deleted this file and the associated package script `validate-init` - we deprecated it [4 years ago](https://github.com/plotly/dash-cytoscape/blob/master/CHANGELOG.md#changed-3) so I think we can 🔪 here 😅